help/en/html/hardware/sprog/SPROG.shtml

```diff
@@ -75,13 +75,20 @@
       <h3>Wiring</h3>
 
       <p>Original SPROGs were connected via a 9-pin serial cable.
-      Current SPROGs are available as USB only. Connect your SPROG
-      using the appropriate cable and power it up.</p>
+      Current SPROGs are available as USB only.<br>
+      Connect your SPROG using the appropriate cable and power it up.</p>
 
       <h3>Settings</h3>
 
       <ol>
-        <li>USB-connected SPROGs on Windows may need to have a
+        <li>USB-connected SPROGs on Windows10 will work using the standard
```

Comment:
Microsoft uses "Windows 10" and "Windows 7" with a space between "Windows" and the version.